oauthproxy_test.go

```diff
@@ -2938,3 +2938,114 @@ func TestAuthOnlyAllowedGroups(t *testing.T) {
 		})
 	}
 }
+
+func TestAuthOnlyRedirectSignIn(t *testing.T) {
```

Comment:
```suggestion
func Test_authOnlyRedirectSignIn(t *testing.T) {
```

(to align to the private naming in the package)